client/src/components/mainContent.jsx

```diff
@@ -9,15 +9,29 @@ import Grid from "@material-ui/core/Grid";
 import Card from "@material-ui/core/Card";
 import CardContent from "@material-ui/core/CardContent";
 import CardHeader from "@material-ui/core/CardHeader";
+import Divider from "@material-ui/core/Divider"
+import CreateNewEventDialog from "../components/CreateNewEventDialog"
+
+
 
 const meetings = [
-  { title: "15min", description: "15 minutes meeting" },
-  { title: "30min", description: "30 minutes meeting" },
-  { title: "60min", description: "60 minutes meeting" },
+  { title: "15min", description: "15 minutes meeting"},
+  { title: "30min", description: "30 minutes meeting"},
+  { title: "60min", description: "60 minutes meeting"},
 ];
 
 const MainContent = () => {
   const classes = useStyles();
+
+  const [open, setOpen] = React.useState(false);
+  const OpenCreateMeetingDialog = async (type) => {
+      setOpen(true);
+  };
+  const CloseCreateMeetingDialog = () => {
```

Comment:
instead of using PascalCase, it should be reserved for naming components. since this is a function, i recommend using snakeCase
```suggestion
  const closeCreateMeetingDialog = () => {
```